.github/workflows/pipeline.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 
     strategy:
       matrix:
-        node: [10.12, 10, '12.0', 12, 14, '16.8']
+        node: [12, 14, 16]
```

Comment:
Note that Node 16.9.1 fixes the regression in 16.9.0, but we still want to include newer versions of 16 in the future